integrations/electricity/energinet.js

```diff
@@ -120,15 +120,18 @@ async function connect({ requestToken }, logger) {
   };
 }
 
-async function collect(state, logger) {
+async function collect(state = {}, logger) {
   const { authToken, locationLon, locationLat } = state;
   const accessToken = await getAccessToken(authToken);
   const { meterPointIds, meterPointAddresses } = await getMeteringPoints(accessToken);
 
-  // Fetch from last update. If not available, then fetch data from the last year.
-  const lastCollect = state.lastCollect ? moment(state.lastCollect) : moment().subtract(1, 'years');
+  // Fetch from last successful date. If not available, then fetch data from the last year.
+  const lastFullyCollectedMoment = state.lastFullyCollectedDay
+    ? moment(state.lastFullyCollectedDay)
+    : moment().subtract(1, 'years');
+  logger.logDebug(`Fetching from ${lastFullyCollectedMoment.toISOString()}`);
```

Comment:
@corradio are we certain that the logger is provided ?

When running the source locally it is not for example

Reply:
Good catch @pierresegonne!

<img width="838" alt="Screenshot 2020-10-14 at 13 27 26" src="https://user-images.githubusercontent.com/3296643/95982890-06756a80-0e21-11eb-9c6c-1f544a0c8e79.png">

Reply:
The other integrations assume a logger is provided so in theory one should always be present!
Maybe worth passing console.log as a logger (just like in the playground)?

Reply:
Yup, that would be fine for now (maybe even using `console.debug`, .warn, etc.). Will you do it or create an issue for it? Guess it's currently blocking energinet integration for working as soon as we merge this :)

Reply:
-> https://github.com/tmrowco/bloom/pull/75